src/test/java/histogram/CashierTransactionHistogramTest.java

```diff
@@ -1,36 +1,78 @@
 package histogram;
 
-import domain.POSDetail;
 import org.junit.Before;
 import org.junit.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import utility.ParsingUtility;
 
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Paths;
+
+
 import java.util.HashMap;
-import java.util.List;
+
 import java.util.Map;
 
+import static junit.framework.TestCase.assertEquals;
 import static junit.framework.TestCase.assertTrue;
 
 
 public class CashierTransactionHistogramTest{
 
-    String inputFile = "/Users/aap1018/Documents/workspace/minorityReportCopy/minorityreport/trendsDataDao/java/transactionCompare/src/main/resources/posDetails_03_2016.csv";
+    String inputFile = "/Users/aap1018/Documents/workspace/javaProjectTemplate/src/main/resources/input/txTest.csv";
```

Comment:
use paths starting from src/ - so anyone building this code can have a successful build - this will fail compilation on someone elses computer.